src/order/ideal.lean

```diff
@@ -129,7 +149,7 @@ end⟩
   Note that we cannot use the `is_coatom` class because `P` might not have a `top` element.
 -/
 @[mk_iff] class is_maximal (I : ideal P) extends is_proper I : Prop :=
-(maximal_proper : ∀ J : ideal P, I < J → J.carrier = set.univ)
+(maximal_proper : ∀ {J : ideal P}, I < J → J.carrier = set.univ)
```

Comment:
This will make it easier to work with the unapplied version of `maximal_proper`:
```suggestion
(maximal_proper : ∀ ⦃J : ideal P⦄, I < J → J.carrier = set.univ)
```